### PR TITLE
yaru-theme: init at 19.10.2

### LINKED ISSUE #66698: package request: yaru theme (communitheme) from ubuntu
**Project description**

https://github.com/ubuntu/yaru

**Metadata**

* homepage URL: https://github.com/ubuntu/yaru
* source URL:
* license: lgpl2/lgpl3
* platforms: linux

### PR DESCRIPTION
**Pull request by jD91mZM2**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Closes #66698 (also see #51153).

### CODE REVIEW

File: pkgs/data/themes/yaru/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, meson, sassc, pkg-config, glib, ninja,
+  python3, gtk3, gnome3, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "yaru-${version}";
```
Comment (worldofpeace) [inline pkgs/data/themes/yaru/default.nix]:
```suggestion
  pname = "yaru";
```

File: pkgs/data/themes/yaru/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  version = "19.10.1";
```
Comment (worldofpeace) [inline pkgs/data/themes/yaru/default.nix]: needs update https://github.com/ubuntu/yaru/releases/tag/19.10.2

**Comment by worldofpeace** [comment]
This interestingly uses gnome-session modes and provides a Yaru session.

File: pkgs/data/themes/yaru/default.nix (hunk 1 above)
Comment (jD91mZM2) [reply pkgs/data/themes/yaru/default.nix]: Woah, TIL!

**Comment by jD91mZM2** [comment]
Oh, is that what it's doing? I have to admit that I didn't really understand what I was writing when adding your gnome-related suggestions from the last PR :)

**Comment by romildo** [comment]
Tested locally. Looks good.

**Pull request merged**